models/eos/interfaces/deleted_example_01.txt

```diff
@@ -0,0 +1,35 @@
+
+# Using deleted
+
+# Before state:
+# -------------
+#
+# veos#show running-config | section interface
+# interface Ethernet1
+#    description "Interface 1"
+# !
+# interface Ethernet2
+# !
+# interface Management1
+#    description "Management interface"
+#    ip address dhcp
+# !
+
+- name: Returns interfaces to default parameters
+  eos_interfaces:
+    config:
+      - name: Ethernet1
+    operation: deleted
```

Comment:
possibly naive question, but if I run this task a second time, will it entirely delete Ethernet1? If so, consider adding a note in the comments, something like '(Note: This won't delete the interface itself if any other values are configured for interface)'